go/arrow/array/bufferbuilder.go

```diff
@@ -151,3 +153,112 @@ func (b *bufferBuilder) unsafeAppend(data []byte) {
 	copy(b.bytes[b.length:], data)
 	b.length += len(data)
 }
+
+type multiBufferBuilder struct {
+	refCount  int64
+	blockSize int
```

Comment:
This should be settable so that users (and tests) can configure their own block size
```suggestion
	BlockSize int
```

Reply:
the `multiBufferBuilder` isn't exported itself, so a user wouldn't even have access to the buffer builder in order to set the blocksize. I can add a method to the `BinaryViewBuilder` that would allow setting the blocksize